spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -305,6 +305,27 @@ that it could not until explicitly told that it can clean up by providing a meth
 
 If multiple methods are annotated with the same annotation an arbitrary one will be chosen.
 
+[[config-parameters]]
+==== Configuration parameters
+
+This specification has 1 configuration parameter which is retrieved using the MicroProfile Config principals.
+
+     mp.lra.LRA.value=SUPPORTS
+
+The allowed values for the configuration parameter are `SUPPORTS`, the default value, and `NOT_SUPPORTED`.
+When an invalid value is defined, the implementation MUST prohibit the successful startup of the application
+(e.g. through a runtime exception).
+
```

Comment:
So if we go with a boolean valued property there is no reason to prohibit startup of the application.

Reply:
Indeed, I'll adjust